elements/pfe-toast/src/pfe-toast.scss

```diff
@@ -0,0 +1,118 @@
+@import "../../pfe-sass/pfe-sass";
+$LOCAL: toast;
+
+:host {
+  // local variables
+  --pfe-toast--Top: 50px;
+  --pfe-toast--MaxWidth: 500px;
+  --pfe-toast--MinWidth: 250px;
+  --pfe-toast--Right--offset: 50px; // needed with box-shadow
+  --pfe-toast--Right: calc(-1 * (#{pfe-local(MaxWidth)} + #{pfe-local(Right--offset)}));
+  --pfe-toast__container--BackgroundColor: #{pfe-color(surface--lightest)};
+  // --pfe-toast__container--BoxShadow: pfe-var(box-shadow--lg);
+  --pfe-toast__container--BoxShadow: rgba(3, 3, 3, 0.13) 0px 3px 7px 3px, rgba(3, 3, 3, 0.12) 0px 11px 24px 16px;
+  --pfe-toast__container--Color: #{pfe-color(text)};
+  --pfe-toast__container--Padding: 1rem;
+  --pfe-toast__close--svg--Fill: #{pfe-color(feedback--default--darkest)};
+  --pfe-toast__close--svg--Fill--hover: #151515;
```

Comment:
Is there a theme color that parities this one?

Reply:
Thanks for updating this! I think the only changes left in the sass file are applying variables to the animation and box shadow properties. I tried using the ones you recommended, but the animation was too fast for the toast, and for some reason the box shadow wasn't applying correctly.